hack/.vendor-helpers.sh

```diff
@@ -28,7 +28,7 @@ clone() {
 	case "$vcs" in
 		git)
 			git clone --quiet --no-checkout "$url" "$target"
```

Comment:
Not a problem -- that just means that initially, nothing will be checked out, so our `checkout` below will do the first checkout.